types/stripe/index.d.ts

```diff
@@ -13755,6 +13834,13 @@ declare namespace Stripe {
             retrieve(tokenId: string, response?: IResponseFn<tokens.IToken>): Promise<tokens.IToken>;
         }
 
+        class OAuth extends StripeResource {
+            token(
+                data: oauth.IOAuthAuthorizationCodeTokenRequest | oauth.IOAuthRefreshTokenRequest,
+                response?: IResponseFn<oauth.IOAuthToken>,
+            ): Promise<oauth.IOAuthToken>;
```

Comment:
Can you add the `deauthorize` function as well?

Reply:
@pushplay do you prefer a full jsdoc comment? I'd like to document the parameters of deauthorize  using the same language stripe uses in their documentation.

```ts
            /**
             * When revoking access to an account, you must use an API key that matches the mode—live or test—of the authorization code (which depends on whether the client_id used was production or development).
             *
             * @param {string} client_id The client_id of the application that you'd like to disconnect the account from. The account must be connected to this application.
             * @param {string} stripe_user_id The account you'd like to disconnect from.
             * @param {HeaderOptions} options
             * @param {IResponseFn<oauth.IOAuthDeauthorizationResponse>} [response]
             * @returns {Promise<oauth.IOAuthDeauthorizationResponse>}
             * @memberof OAuth
             */
            deauthorize(
                client_id: string,
                stripe_user_id: string,
                options: HeaderOptions,
                response?: IResponseFn<oauth.IOAuthDeauthorizationResponse>,
            ): Promise<oauth.IOAuthDeauthorizationResponse>;
```

Reply:
nvm i see that it's redundant so i've only documented the `client_id` and `stripe_user_id`